website/src/components/ui/select.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from "react"
```

Comment:
we need a licence header on all new files

Reply:
done in resolved in 
2a64125